### DISCUSSION TITLE
Date format and start of week

### DISCUSSION THREAD

**Discussion by madic-creates** [TriliumNext/Trilium#2908] [General]
Hello,
is it possible to change the format of the date for a task from "mm/dd/yyyy" to "yyyy - mm - dd"
and set the start of the week to monday?
I am using the "task template" from the trilium demo.
The attribute todoDate has the type Date. But I can't find a setting to change the date format

**Comment by zadam** [comment]
Hi, the input is a native electron/chrome/HTML widget and it's not configurable like that (AFAIK), it always uses date according to user's locale.

**Comment by madic-creates** [comment]
Do you know the required environment variables or files from which electron is getting the information? Because I've set the following env variables:

```
LC_ADDRESS=de_DE.UTF-8
LC_IDENTIFICATION=de_DE.UTF-8
LC_MEASUREMENT=de_DE.UTF-8
LC_MONETARY=de_DE.UTF-8
LC_NAME=de_DE.UTF-8
LC_NUMERIC=de_DE.UTF-8
LC_PAPER=de_DE.UTF-8
LC_TELEPHONE=de_DE.UTF-8
LC_TIME=de_DE.UTF-8
TZ=Europe/Berlin
```

**Reply by FliegendeWurst** [reply]
Electron reads the `LANGUAGE` environment variable.

**Reply by madic-creates** [reply]
Thank you. That helped me
